src/components/ModularTable/ModularTable.tsx

```diff
@@ -97,9 +105,14 @@ function ModularTable({
                       : "",
                   },
                   { ...getHeaderProps?.(column) },
+                  // Only call this if we want it to be sortable too.
+                  sortable ? column.getSortByToggleProps() : undefined,
                 ])}
               >
                 {column.render("Header")}
+                <span>
+                  {column.isSorted ? (column.isSortedDesc ? " 🔽" : " 🔼") : ""}
```

Comment:
We shouldn't introduce new elements and emoji in the table UI.

Vanilla has it's own built-in styles for sortable tables: https://vanillaframework.io/docs/base/tables#sortable
Technically all it needs it adding `aria-sort` attribute on the `th` element (with one of the values of "none" "ascending" "descending"), so ideally `column.isSorted` should affect `aria-sort` on header, not add content to it.